assets/src/blocks/Covers/deprecated/coversV1.js

```diff
@@ -0,0 +1,39 @@
+export const coversV1 = {
+  attributes: {
+    title: {
+      type: 'string',
+    },
+    description: {
+      type: 'string',
+    },
+    tags: {
+      type: 'array',
+      default: []
+    },
+    posts: {
+      type: 'array',
+      default: []
+    },
+    post_types: {
+      type: 'array',
+      default: []
+    },
+    covers_view: {
+      type: 'string',
+      default: '1'
+    },
+    cover_type: {
+      type: 'string',
+    },
+  },
+  isEligible(attributes) {
+    return attributes.covers_view;
+  },
+  migrate( { covers_view, ...attributes } ) {
+    attributes.initialRowsLimit = covers_view === '3' ? 0 : Number(covers_view);
+    delete attributes.covers_view;
```

Comment:
No need to delete it, it's already the other attributes because of `{ covers_view, ...attributes }`